test/sagemaker_tests/pytorch/inference/integration/local/test_serving.py

```diff
@@ -50,6 +53,24 @@
 }
 
 
+def disable_token_auth(framework_version):
```

Comment:
Also can we move this function to `test/sagemaker_tests/pytorch/inference/integration/__init__.py` file? That we we're declaring it in one file and then we can import the functions across multiple test files.